src/python/pants/backend/python/lint/black/rules.py

```diff
@@ -32,16 +32,21 @@
 class BlackSetup:
   """This abstraction is used to deduplicate the implementations for the `fmt` and `lint` rules,
   which only differ in whether or not to append `--check` to the Black CLI args."""
-  config_path: Optional[Path]
   resolved_requirements_pex: Pex
+  config_path: Optional[Path]
+  wrapped_target: FormattablePythonTarget
```

Comment:
I don't think the setup class should know about the target - a "setup" should be shared across all targets. However it should know about the passthrough_args (or just "args" now, I don't think the "passthrough" verbiage is helpful any more) as they are fixed across all targets.